.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: '17'
+          node-version: '18'
```

Comment:
Why did we change the node version here?

Reply:
I changed it to match the version in the backend. You can find the explanation for the change in the backend in the corresponding PR.